forch/config_file_watcher.py

```diff
@@ -0,0 +1,98 @@
+"""Faucet config file watcher"""
+
+import hashlib
+import logging
+import os
+import yaml
+
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+
+LOGGER = logging.getLogger('watcher')
+
+
+class ConfigFileWatcher:
+    """Watch file changes"""
+    def __init__(self, structural_config_file, faucetizer):
```

Comment:
this is suffering from dependency inversion -- this class shouldn't know anything about "faucetizer" or faucetizer specific logic (the _reschedule_acl_file_handlers function). Rather, it should accept lambda functions that perform the callback, and then the faucetizer specific logic should be handled outside of this class.

Reply:
Moved _reschedule_acl_file_handlers to forchestrator; passing faucetizer logic as callback